### ISSUE TITLE
Provide the option to WAIT on a VSCODE.diff in a TreeItem extension

### ISSUE THREAD

**Issue by davidkjackson54** [microsoft/vscode#112236]
- VSCode Version: 1.51.1
- OS Version: WIndows 10

I have a VSCode Treeview extension that is so far working fine. I offer a right click on a TreeItem node to perform a vscode.diff before proceeding with an update function. 
I call a function:
`await diffCompare(originalFile, developmentFile);

const dj = "x";`

and the function itself:

`export async function diffCompare(original: string, current: string) {

await vscode.commands.executeCommand("vscode.diff", vscode.Uri.file(original), vscode.Uri.file(current));

return;
}`

I am finding that it is dropping straight to the breakpoint that I set instead of remaining in the diff till it is complete and the files closed. 

I am told that this is not doable without the means to provide an enhancement to provide a "WAIT" option to the vscode.diff.

Can this be put forward?

**Comment by alexr00** [comment]
@bpasero do you own `vscode.diff` ?

**Comment by bpasero** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/112183
